public/app/plugins/datasource/elasticsearch/plugin.json

```
@@ -3,27 +3,29 @@
   "name": "Elasticsearch",
   "id": "elasticsearch",
   "category": "logging",
-
   "info": {
     "description": "Open source logging & analytics database",
     "author": {
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
-    "keywords": ["elasticsearch"],
+    "keywords": ["elasticsearch", "datasource", "database", "logs", "metrics"],
```

I realize i wasn't very clear, but i listed logs and metrics as examples, based on opensearch i suspect we should be adding these instead:   https://github.com/grafana/opensearch-datasource/blob/a66f595e6be35a3b131b234ffd32b998152bcb1f/src/plugin.json#L19
```suggestion
    "keywords": ["elasticsearch", "datasource", "database", "logs", "nosql", "traces"],
```

---

ah whoops, makes sense. Updated in latest commit